code-challenges.test.js

```diff
@@ -14,54 +14,128 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const studyMinutesWeek1 = {
-  sunday: 90,
-  monday: 30,
-  tuesday: 20,
-  wednesday: 15,
-  thursday: 30,
-  friday: 15,
-  saturday: 60
-}
-// Expected output: [15, 15, 20, 30, 30, 60, 90]
-
-const studyMinutesWeek2 = {
-  sunday: 100,
-  monday: 10,
-  tuesday: 45,
-  wednesday: 60,
-  thursday: 20,
-  friday: 15,
-  saturday: 65
-}
-// Expected output: [10, 15, 20, 45, 60, 65, 100]
+describe("arraySorter", () => {
+  const studyMinutesWeek1 = {
+    sunday: 90,
+    monday: 30,
+    tuesday: 20,
+    wednesday: 15,
+    thursday: 30,
+    friday: 15,
+    saturday: 60
+  }
+  // Expected output: [15, 15, 20, 30, 30, 60, 90]
+  const studyMinutesWeek2 = {
+    sunday: 100,
+    monday: 10,
+    tuesday: 45,
+    wednesday: 60,
+    thursday: 20,
+    friday: 15,
+    saturday: 65
+  }
+  // Expected output: [10, 15, 20, 45, 60, 65, 100]
+  it("returns an array of the values sorted from least to greatest", () => {
+    expect(arraySorter(studyMinutesWeek1)).toEqual([15, 15, 20, 30, 30, 60, 90])
+    expect(arraySorter(studyMinutesWeek2)).toEqual([10, 15, 20, 45, 60, 65, 100])
+  })
+})
+
+// ReferenceError: arraySorter is not defined
 
 // b) Create the function that makes the test pass.
 
+// pseudo: (this one is very bloated again lol, learned a lot about .sort() this time and wanted to share my new knowledge :) )
+// Create a function named arraySorter that takes an object as an argument
+// ultimately we want our function to return an array of the 'values' of our object argument sorted from lowest to highest value.
+// to do this we need to define a variable named newArray within our function that creates an array from the values contained within the argument object. Object.values() is a built-in method that takes an object as an argument and returns an array of the argument objects values. We need to pass our argument object for our arraySorter function into this built-in method and assign it to newArray.
+// .sort() is a built in method that sorts an array in ascending order. However, while .sort() is a useful tool, they way it translates and defines what value is greater than another can be confusing, especially when dealing with numbers. this built-in method converts each element into a string, then sorts those strings by their Unicode order. Briefly, this means that .sort() will sort 65 before 7 because '6' is before '7', even though 7 is less than 65. so we have to define within our .sort() how we want to sort our values by passing in a function. .sort((a, b) => (a - b)) forces our method to compare values as numbers. if after two values are compared the result is negative, 'a' will be placed before 'b'. if the result is 0, the order will not change, and if the result is a positive value, 'b' will be placed before 'a'.
+// 
+// put a return on that bad boy, and you've got yourself a function that sorts the values from an object!
```

Comment:
Awesome job on this one! I love the pseudocode being bloated and you taking the time to really look into how something works.